server/app/models/AccountModel.java

```diff
@@ -187,4 +195,42 @@ public String getApplicantDisplayName() {
         .map(u -> u.getApplicantData().getApplicantDisplayName().orElse("<Unnamed User>"))
         .orElse("<Unnamed User>");
   }
+
+  public Optional<SessionDetails> getActiveSession(String sessionId) {
+    return activeSessions.containsKey(sessionId)
```

Comment:
**suggestion:** You could also consider doing `Optional.ofNullable(activeSessions.get(sessionId))`